<ui version="4.0" >
 <class>DisplayPopulationWidgetUI</class>
 <widget class="QWidget" name="DisplayPopulationWidgetUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DisplayPopulationWidgetUI</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QRadioButton" name="_show_selected" >
     <property name="text" >
      <string>Show selected</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QRadioButton" name="_show_fittest" >
     <property name="text" >
      <string>Show fittest</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QRadioButton" name="_show_all" >
     <property name="text" >
      <string>Show all</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QFrame" name="_display" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
